scripts/commands/getstats.lua

```diff
@@ -4,14 +4,22 @@
 -----------------------------------
 local commandObj = {}
 
+local options =
+{
+    'base',
+    'defensive',
+    'offensive',
+    'elements',
+}
+
 commandObj.cmdprops =
 {
     permission = 1,
-    parameters = ''
+    parameters = 'i'
```

Comment:
why not just use string parameter and make default no-param result be to tell the user what the potential parameters are and return out (other command use an error handler wrapper) instead of looping your table to print them?

Reply:
simpler to remember an integer rather than a stat group name

Reply:
`['string'] = number,` / `[number] = 'string',` and you can have it both ways

Reply:
table `allthethings.gif`